From ef01b95081981c7f72665f6a3797ceb686392dca Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 1 Nov 2012 14:37:25 +0100
Subject: [PATCH 24/40] Let v4l2sink use V4L2_BUF_TYPE_VIDEO_OUTPUT_OVERLAY

V4L2_BUF_TYPE_VIDEO_OVERLAY is reserved for capture overlays
that don't need to be fed with data from userspace.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 sys/v4l2/gstv4l2sink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2sink.c b/sys/v4l2/gstv4l2sink.c
index c3f6e7f..10ad2eb 100644
--- a/sys/v4l2/gstv4l2sink.c
+++ b/sys/v4l2/gstv4l2sink.c
@@ -285,7 +285,7 @@ gst_v4l2sink_sync_overlay_fields (GstV4l2Sink * v4l2sink)
     struct v4l2_format format;
 
     memset (&format, 0x00, sizeof (struct v4l2_format));
-    format.type = V4L2_BUF_TYPE_VIDEO_OVERLAY;
+    format.type = V4L2_BUF_TYPE_VIDEO_OUTPUT_OVERLAY;
 
     if (v4l2_ioctl (fd, VIDIOC_G_FMT, &format) < 0) {
       GST_WARNING_OBJECT (v4l2sink, "VIDIOC_G_FMT failed");
-- 
1.8.3

